lib/charms/traefik_k8s/v1/ingress_per_unit.py

```diff
@@ -801,7 +801,7 @@ def _urls_from_relation_data(self) -> Dict[str, str]:
         if not relation:
             return {}
 
-        if not all((relation.app, relation.app.name)):  # type: ignore
+        if not all([r for r in [relation.app, relation.app.name]]):  # type: ignore
```

Comment:
I can easily imagine someone refactoring out the "redundancy".

Reply:
I see what you're doing but IMHO `if relation.app and relation.app.name` is so much more clear